### ISSUE TITLE
False "Your account has been flagged for being associated with a smurf account" Flag

### ISSUE THREAD

**Issue by ValveHatesDotA** [ValveSoftware/Dota2-Gameplay#11289]
### Description

I don't have a smurf account but I still received this warning. I have two accounts but alternate account =/= smurf account, both are pretty much in the same bracket. I received the same warning on both of these accounts though so clearly it's falsely associating these accounts with another account that doesn't exist.

Fix this shit. Don't roll out awful, broken systems. @jeffhill 

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21988932/10de71d7-16d8-40b1-975e-853eab550977)

**Comment by ValveHatesDotA** [comment]
Nice downvote abuse. This isn't reddit losers

**Comment by TobyFlendersonFromHR** [comment]
I believe in you

**Comment by ValveHatesDotA** [comment]
> I believe in you

I don't care about what you believe.

**Comment by TobyFlendersonFromHR** [comment]
I'm glad both your accounts got flagged.

**Comment by ValveHatesDotA** [comment]
> I'm glad both your accounts got flagged.

Cool. Your accounts would never get flagged since you're not even good enough to win vs drunk heralds

**Comment by TobyFlendersonFromHR** [comment]
Jokes on you, i can't be flagged because i don't play this game at all

**Comment by ValveHatesDotA** [comment]
> Jokes on you, i can't be flagged because i don't play this game at all

You do, you've been hardstuck in terms of skill for the last 8 years

**Comment by TobyFlendersonFromHR** [comment]
No, I literally don't play at all. Imagine being stupid enough to play this game like you do, shame.
